pkg/agent/datamodel/types.go

```diff
@@ -336,6 +337,18 @@ type WindowsProfile struct {
 	EnableAHUB                    *bool             `json:"enableAHUB,omitempty"`
 	WindowsPauseImageURL          string            `json:"windowsPauseImageURL"`
 	AlwaysPullWindowsPauseImage   *bool             `json:"alwaysPullWindowsPauseImage,omitempty"`
+	WindowsRuntimes               *WindowsRuntimes  `json:"windowsRuntimes,omitempty"`
+}
+
+// WindowsRuntimes configures containerd runtimes that are available on the windows nodes
```

Comment:
i'm confused , is this windows runtime or containerd runtime?

Reply:
@AbelHu why we did not use container runtime but windows runtime instead? To differentiate from the existing ContainerRuntime? Or we may consider moving this to WindowsContainerRuntime?

Reply:
This is windows runtime for hyperv isolation. For example, when it is set 17763 with using 2004, it means to run a 2019 hyper-v VM in a 2004 host.

Reply:
please update the comments then. "WindowsRuntimes configures containerd runtimes" is where i'm confused